### PR TITLE
Bug 1171831 - Confusing footer for “Settings” page

### PR DESCRIPTION
**Pull request by atiqueahmedziad**

### CODE REVIEW

File: src/templates/footer.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h1 class="footzilla"><a href="https://www.mozilla.org/" target="_blank">mozilla
            legal_url='http://www.mozilla.org/about/legal.html#site',
            cc_url='http://creativecommons.org/licenses/by-sa/3.0/') }}
     </p>
-    <ul class="footer-extras">
+    <ul class="footer-extras hide-on-tablet">
```
Comment (mstriemer) [inline src/templates/footer.html]: I don't think we want to change this.

File: src/templates/settings.html [hunk 2]
```diff
@@ -63,7 +63,7 @@
     </div>
   </section>
 
-  <section class="main full hide-on-desktop account-settings-links">
```
Comment (mstriemer) [inline src/templates/settings.html]: This should change to `hide-on-tablet` instead of just removing `hide-on-desktop`.

Comment (atiqueahmedziad) [inline src/templates/settings.html]: Here there is reviewer tools button. So i thought if a reviewer wants to use these in their tablet,  they should have these buttons. Rather we can hide the `footer-extras` links as we have the privacy,  terms button available here along with get involved button & reviewer button. What do you think? :)

Comment (mstriemer) [inline src/templates/settings.html]: I think if you're a reviewer you get a link to the reviewer tools in the footer on each page. I think the footer should stay as-is, it was setup that way for a reason. It seems to me like the settings links are just there for when the footer isn't visible so we should hide the settings links instead.

Comment (atiqueahmedziad) [inline src/templates/settings.html]: Ok :) thanks for the info. Will edit it tomorrow :)

**Comment by atiqueahmedziad** [comment]
@mstriemer  review pls?  :)

**Comment by mstriemer** [comment]
Looks good, thanks for the patch!

**Pull request merged**